docs/limitations.md

```diff
@@ -22,6 +22,11 @@ featureMaturity:
     `spark.mesos.[driver|executor].secret.envkeys, the keystore and truststore secrets will also show up as
     environment-based secrets, due to the way secrets are implemented. You can ignore these extra environment variables.
     
+*   Anyone who has access to the Spark service has access to all secrets
```

Comment:
I would add a **WARNING** to the beginning of this (or some equivalent markup that == "HEY PAY ATTENTION!")

Reply:
Will do